draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -326,14 +306,27 @@ Link: <https://example.org/dict.dat>; rel="compression-dictionary"
 
 # Dictionary-Compressed Brotli
 
-The "br-d" content encoding identifies a resource that is a
+The "dcb" content encoding identifies a resource that is a
 "Dictionary-Compressed Brotli" stream.
 
-A "Dictionary-Compressed Brotli" stream is a Brotli {{RFC7932}} stream for a
-response that has been compressed with an external dictionary using a
-compression window not larger than 16 MB.
+A "Dictionary-Compressed Brotli" stream is a binary stream that starts with a
+36-byte header and is followed by a Brotli {{RFC7932}} stream of the response
+that has been compressed with an external dictionary using a compression window
+not larger than 16 MB.
+
+The 36-byte stream header consists of a 4-byte signature followed by the
+32-byte SHA-256 hash of the external dictionary that was used to compress the
+resource:
+
+|  Field             | Size       |
+|--------------------|------------|
+|  Magic_Number      | 4 bytes    |
+|  SHA_256_Hash      | 32 bytes   |
```

Comment:
I'm not sure if the table really gets you much.

You might instead consider using a definition list, as I suggest below.